lib/node_modules/@stdlib/lapack/base/dlarfb/README.md

```diff
@@ -0,0 +1,270 @@
+
+<!-- eslint-disable max-len -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var V = new Float64Array( [ 10.0, 40.0, 70.0, 20.0, 50.0, 80.0, 30.0, 60.0, 90.0 ] );
+var T = new Float64Array( [ 1.0, 0.0, 0.0, 0.0, 2.0, 0.0, 0.0, 0.0, 3.0 ] );
+var C = new Float64Array( [ 11.0, 12.0, 13.0, 21.0, 22.0, 23.0, 31.0, 32.0, 33.0 ] );
+var work = new Float64Array( 9 );
+
+dlarfb.ndarray( 'left', 'transpose', 'forward', 'columns', 3, 3, 3, V, 3, 1, 0, T, 3, 1, 0, C, 3, 1, 0, work, 3, 1, 0 );
+// C => <Float64Array>[ -1350.00, -1400.00, -1450.00, -30961.00, -32102.00, -33243.00, -266612.00, -275464.00, -284316.00 ]
+```
+
+The function has the following additional parameters:
+
+-   **side**: specifies whether `H` or `H ^ T` is applied from the left or right (`'left'` or `'right'`).
+-   **trans**: specifies whether to apply `H` or `H ^ T` (`'no-transpose'` or `'transpose'`).
+-   **direct**: indicates how `H` is formed from a product of elementary reflectors (`'forward'` or `'backward'`).
+-   **storev**: indicates how the vectors which define the elementary reflectors are stored (`'columns'` or `'rows'`).
+-   **M**: number of rows of the matrix `C`.
+-   **N**: number of columns of the matrix `C`.
+-   **K**: order of the matrix `T`.
+-   **V**: input matrix stored as a [`Float64Array`][mdn-float64array].
+-   **strideV1**: stride of the first dimension of `V`.
+-   **strideV2**: stride of the second dimension of `V`.
+-   **offsetV**: index offset for `V`.
+-   **T**: input matrix stored as a [`Float64Array`][mdn-float64array].
+-   **strideT1**: stride of the first dimension of `T`.
+-   **strideT2**: stride of the second dimension of `T`.
+-   **offsetT**: index offset for `T`.
+-   **C**: input/output matrix stored as a [`Float64Array`][mdn-float64array].
+-   **strideC1**: stride of the first dimension of `C`.
+-   **strideC2**: stride of the second dimension of `C`.
+-   **offsetC**: index offset for `C`.
+-   **work**: work array stored as a [`Float64Array`][mdn-float64array].
+-   **strideWork1**: stride of the first dimension of `work`.
+-   **strideWork2**: stride of the second dimension of `work`.
+-   **offsetWork**: index offset for `work`.
+
+While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameters support indexing semantics based on starting indices.
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   Both functions apply the block reflector `H` to matrix `C` according to the specified parameters.
+-   The `side` parameter determines whether the transformation is applied from the left or right.
+-   The `trans` parameter determines whether to apply `H` or its transpose `H^T`.
+-   The `direct` parameter indicates the order in which elementary reflectors are applied.
+-   The `storev` parameter indicates how the elementary reflector vectors are stored.
```

Comment:
In fact, these notes also seem to repeat what is written above in the parameter descriptions.